examples/vision/mixUp.py

```diff
@@ -0,0 +1,218 @@
+"""
+Title: mixup augmentation for image classification
+Author: [Sayak Paul](https://twitter.com/RisingSayak)
+Date created: 2021/03/06
+Last modified: 2021/03/06
+Description: Data augmentation using the mixup technique for image classification.
+"""
+"""
+## Introduction
+"""
+
+"""
+mixup is a *domain-agnostic* data augmentation technique proposed in [mixup: Beyond
+Empirical Risk Minimization](https://arxiv.org/abs/1710.09412) by Zhang et al. It's
+implemented with the following formulas:
```

Comment:
Rather than just the formula, it would be useful to also explain the intuition behind the technique, and explain when/why to use it (as opposed to other augmentation techniques)

Reply:
Included.